manifests/0000_09_cluster-osin-operator_08_auth_cr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+items:
```

Comment:
uh just realized this is a list instead of a single item, same with other CR

Reply:
: ) /hold

Reply:
yup, cluster is much happier now,  i guess it wasn't the flaky CI, it was my flakiness, time for a break :)